includes/CLI/Plugin_List_Checks_Command.php

```diff
@@ -0,0 +1,163 @@
+<?php
+/**
+ * Class WordPress\Plugin_Check\CLI\Plugin_List_Checks_Command
+ *
+ * @package plugin-check
+ */
+
+namespace WordPress\Plugin_Check\CLI;
+
+use WordPress\Plugin_Check\Checker\Default_Check_Repository;
+use WordPress\Plugin_Check\Plugin_Context;
+use WP_CLI;
+
+/**
+ * Plugin list checks command.
+ */
+final class Plugin_List_Checks_Command {
+
+	/**
+	 * Plugin context.
+	 *
+	 * @since n.e.x.t
+	 * @var Plugin_Context
+	 */
+	protected $plugin_context;
+
+	/**
+	 * Output format type.
+	 *
+	 * @since n.e.x.t
+	 * @var string[]
+	 */
+	protected $output_formats = array(
+		'table',
+		'csv',
+		'json',
+	);
+
+	/**
+	 * Constructor.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param Plugin_Context $plugin_context Plugin context.
+	 */
+	public function __construct( Plugin_Context $plugin_context ) {
+		$this->plugin_context = $plugin_context;
+	}
+
+	/**
+	 * Runs plugin list-checks.
+	 *
+	 * ## OPTIONS
+	 *
+	 * [--fields=<fields>]
+	 * : Limit displayed results to a subset of fields provided.
+	 *
+	 * [--format=<format>]
+	 * : Format to display the results. Options are table, csv, and json. The default will be a table.
+	 * ---
+	 * default: table
+	 * options:
+	 *   - table
+	 *   - csv
+	 *   - json
+	 * ---
+	 *
+	 * @subcommand list-checks
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param array $args       List of the positional arguments.
+	 * @param array $assoc_args List of the associative arguments.
+	 *
+	 * @throws WP_CLI\ExitException Show error if not valid runner.
+	 *
+	 * @SuppressWarnings(PHPMD.NPathComplexity)
+	 * @SuppressWarnings(PHPMD.ExcessiveMethodLength)
+	 */
+	public function list_checks( $args, $assoc_args ) {
+		$check_repo = new Default_Check_Repository();
+
+		$all_checks = array();
+
+		$checks_items = $check_repo->get_checks()->to_map();
+
+		if ( is_array( $checks_items ) && count( $checks_items ) > 0 ) {
+			foreach ( $checks_items as $key => $check ) {
+				$item = array();
+
+				$item['slug']      = $key;
+				$item['stability'] = strtolower( $check->get_stability() );
+				$item['category']  = join( ', ', $check->get_categories() );
+
+				$all_checks[] = $item;
+			}
+		}
```

Comment:
No need for the if clause, without it the code would work just the same.
```suggestion
		foreach ( $checks_items as $key => $check ) {
			$item = array();

			$item['slug']      = $key;
			$item['stability'] = strtolower( $check->get_stability() );
			$item['category']  = join( ', ', $check->get_categories() );

			$all_checks[] = $item;
		}
```